packages/http-client-java/generator/http-client-generator/src/main/java/com/microsoft/typespec/http/client/generator/model/EmitterOptions.java

```diff
@@ -22,7 +22,7 @@ public class EmitterOptions implements JsonSerializable<EmitterOptions> {
     private List<String> serviceVersions;
     private Boolean generateTests = true;
     private Boolean generateSamples = true;
-    private Boolean enableSyncStack = true;
+    private Boolean enableSyncStack;
```

Comment:
Should only affect MPG.

For DPG, we have logic to set enableSyncStack to true by default, then override if this configuration exists.

For MPG, currently we disable it.